message_ix/core.py

```diff
@@ -119,8 +119,12 @@ def vintage_and_active_years(self):
         return v_years, a_years
 
     def solve(self, **kwargs):
-        """Solve a MESSAGE Scenario. See ixmp.Scenario.solve() for arguments"""
-        return super(Scenario, self).solve(model='MESSAGE', **kwargs)
+        """Solve a MESSAGE Scenario. See ixmp.Scenario.solve() for arguments.
+        MESSAGE is provided by default, but can be overwritten with, e.g.,
```

Comment:
suggest to add quotation marks, and rephrase to "The default model is 'MESSAGE', but can ...".
It's not quite intuitive what "MESSAGE is provided..." should mean.